create-prs-for-vcs-repositories-update/create_prs_for_vcs_repositories_update.py

```diff
@@ -0,0 +1,333 @@
+import os
+import re
+import argparse
+import logging
+from typing import Optional
+
+from ruamel.yaml import YAML
+from packaging import version
+import git
+import github
+from github import Github    # cspell: ignore Github
+
+
+# Parse arguments
+parser = argparse.ArgumentParser(description="Create a PR to update version in autoware.repos")
+
+# Verbosity count
+parser.add_argument("-v", "--verbose", action="count", default=0, help="Verbosity level")
+
+# Repository information
+args_repo = parser.add_argument_group("Repository information")
+args_repo.add_argument("--parent_dir", type=str, default="./", help="The parent directory of the repository")
+args_repo.add_argument("--repo_name", type=str, default="autowarefoundation/autoware_dummy_repository", help="The repository name to create a PR")
+args_repo.add_argument("--base_branch", type=str, default="main", help="The base branch of autoware.repos")
+args_repo.add_argument("--new_branch_prefix", type=str, default="feat/update-", help="The prefix of the new branch name")
+
+'''
+Following default pattern = r'\b(v?\d+\.\d+(?:\.\d+)?(?:-\w+)?(?:\+\w+(\.\d+)?)?)\b'
+can parse the following example formats:
+    "0.0.1",
+    "v0.0.1",
+    "ros2-v0.0.4",
+    "xxx-1.0.0-yyy",
+    "2.3.4",
+    "v1.2.3-beta",
+    "v1.0",
+    "v2",
+    "1.0.0-alpha+001",
+    "v1.0.0-rc1+build.1",
+    "2.0.0+build.1848",
+    "2.0.1-alpha.1227",
+    "1.0.0-alpha.beta",
+    "ros_humble-v0.10.2"
+'''
+args_repo.add_argument("--semantic_version_pattern",
+                       type=str,
+                       default=r'\b(v?\d+\.\d+(?:\.\d+)?(?:-\w+)?(?:\+\w+(\.\d+)?)?)\b',
+                       help="The pattern of semantic version"
+)
+
+# For the Autoware
+args_aw = parser.add_argument_group("Autoware")
+args_aw.add_argument("--autoware_repos_file_name", type=str, default="autoware.repos", help="The path to autoware.repos")
+
+args = parser.parse_args()
+
+
+# Initialize logger depending on the verbosity level
+if args.verbose == 0:
+    logging.basicConfig(level=logging.WARNING)
+elif args.verbose == 1:
+    logging.basicConfig(level=logging.INFO)
+elif args.verbose >= 2:
+    logging.basicConfig(level=logging.DEBUG)
+
+
+logger = logging.getLogger(__name__)
```

Comment:
Please move them after `if __name__ == "__main__"`.

Can you make `parse_args` function?
https://github.com/tier4/autoware-ml/blob/28cd951e5d7993896aff0c8c053908a799b47edb/tools/detection3d/visualize_bev.py#L94

Reply:
I appreciate your proposal. Now applied by [this commit](https://github.com/autowarefoundation/autoware-github-actions/pull/317/commits/f7546c68c8aad92193a3b093d0b589db7c9e60f1).